### DISCUSSION TITLE
Suggestion - Turreted Weapon Controls Toggle

### DISCUSSION THREAD

**Discussion by zalexville** [tsunamayo/Starship-EVO#4064] [Suggestions]
Good morning/afternoon,

I currently have been working on a build where I would prefer to have all the primary turrets control by the flight joystick. Currently I have to set the turrets to target based off ship targetting rather than myself being able to point and click. 
When it is set to point and click, I have to leave the joystick to move my wings in which the turrets are mounted before I can then control them, otherwise the wings freeze up and get stuck.

Could we get built-in keybind (like how landing gear (L) will be with the rumored removal of the hotkey logic) to enable or disable whether we are currently controlling the turrets? 

I realy appreciate all the work you have done to the game over the past couple years and I am excited to see the game keep getting better and better!

**Comment by tsunamayo** [comment]
@zalexville Hi, I am not sure to understand, you want a way to no longer control the turret - while still controlling your ship?
Why cant you control your ship, then enter another yoke linked to the turret when you wish to?
For sure I wont add a hotkey for such a specific case - but you will be able to do more powerful stuff in the future (like overwriting the config of the turret with a button).

**Comment by zalexville** [comment]
Sorry I'm bad at describing it. Basically I have a yoke to fly the ship, but I also wanted to have turrets controlled through it so the pilot can dumbfire (without needing to have a locked target). I have the guns mounted to the wings of my BT-7 Thunderclap (The blueprint is in my other bug report (#4061). When I try to open the wings while control the ship inthe yoke, they cant open. But if I leave the seat and stop controlling the ship and guns the wings can open normally. 

I was wondering if there could be a key to enable or disable controlling the turrets without the need of another yoke or to have to always have the turrets set to starship target. 

If you want to test for yourself to get a better idea of what I'm poorly explaining, its yoke on the left side and the bright blue button on the center console to open/close the wings.
